src/main/java/org/terasology/signalling/componentSystem/SignalSwitchBehaviourSystem.java

```diff
@@ -195,19 +198,36 @@ public void handleDelayedTrigger(String actionId, EntityRef entity) {
                 });
     }
 
+    /**
+     * Updates the SignalSwitchBehaviorSystem.
```

Comment:
For this one you don't have a lot to work with since the method just calls two other methods. The doc *seems* like it is simply duplicating the methods inside `update` by just describing their names in text, and a developer reading the full method and the doc wouldn't really learn more from it. 

However, somebody just reading the javadoc solo could still learn from it without the code available. So even though it seems somewhat duplicate-y it could be valuable.

Only real alternative I could think of would be something like "Handles game loop ticks for signal switches" but I'm not really sure if that's better or worse.